config/management/operational/src/command.rs

```diff
@@ -118,7 +118,7 @@ impl std::fmt::Display for CommandName {
             CommandName::RemoveValidator => "remove-validator",
             CommandName::RotateConsensusKey => "rotate-consensus-key",
             CommandName::RotateOperatorKey => "rotate-operator-key",
-            CommandName::RotateFullNodeNetworkKey => "rotate-fullnode-network-key",
+            CommandName::RotateFullNodeNetworkKey => "rotate-full-node-network-key",
```

Comment:
if you do this, you're going to have to update the scripts and the runbooks.....

Reply:
I think

Reply:
So there's two points to make here:
- The existing runbooks don't actually talk about this command at all. So we need to improve our runbooks.
- The command is actually "rotate-full-node-network-key" (note: RotateFullNodeNetworkKey -- "Node" has a capital, so this is currently a bug which wasn't detected because there was no smoke test for it). I don't mind which we use, so I chose the variant with the smallest number of code changes. Overall, it's probably worth pointing out that there's a lot of places in our codebase where we use "full node" and "fullnode". It's not great and it's kinda ugly. Perhaps we should discuss this with the networking folks and settle on something so we can clean up the codebase?